include/swift/AST/DiagnosticsParse.def

```diff
@@ -217,6 +217,7 @@ ERROR(let_cannot_be_addressed_property,none,
 ERROR(disallowed_var_multiple_getset,none,
       "'var' declarations with multiple variables cannot have explicit"
       " getters/setters", ())
+NOTE(found_unexpected_stmt,none, "found an unexpected statement here", ())
```

Comment:
Once you’re done adding new diagnostics, please copy them to the `en.yaml` diagnostic file as well.

Reply:
Are we duplicating all the diagnostics between the two files? Is this expectation documented somewhere?

Reply:
We'd have to do this for now unfortunately, we are trying to get to the place where `.def` files would no longer contain diagnostic messages only ids and argument types.

Reply:
Also just to make it clear - this is not at all a hard requirement, we'll audit everything while removing strings from `.def`, mirroring diagnostic changes would help us out though.

Reply:
Should the diagnostic message be in both places for now? Or should I take the string out of `DiagnosticsParse.def` leaving just the code, and put the string in the localization yaml?

Reply:
Diagnostic message should be in both places for now.